modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -638,6 +638,18 @@ definition {
 			locator1 = "ObjectAdmin#RELATIONSHIP_LABEL");
 	}
 
+	macro assertObjectSortedEntry {
+		var entrySort = JSONObject.getObjectSortedEntry(
+			objectName = "${objectName}",
+			sortString = "${sortString}");
+
+		var entrySortValue = JSONUtil.getWithJSONPath("${entrySort}", "$..items..${sortString}");
+
+		TestUtils.assertEquals(
+			actual = "${entrySortValue}",
+			expected = "${expectedValue}");
+	}
+
```

Comment:
@Luizcarlosqueiroz can you delete this macro and move it to the test level? We shouldn't be making assertions in macros and since we aren't really reusing this, there isn't a need to have a macro in the first place.

Reply:
@timpak I can change the assertion to be out of the macro, but I was going to leave the API call and the JSONUtil at the macro layer. Do you think it works this way?

Reply:
Whoops, I didn't see this, yeah that's fine.